<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppercuAlgorithmeWidget</class>
 <widget class="QWidget" name="AppercuAlgorithmeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>439</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Algorithme</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="GlobalverticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="labeldescriptionHorizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelDescription">
         <property name="text">
          <string> DÃ©scription  de l'algorithme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCommentaireDescription">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditDescription">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="LabelCodeHorizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCodeAlgo">
         <property name="text">
          <string>  Code de l'algorithme </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCommentaireCodeAlgo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeViewCodeAlgo">
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
